cost-analyzer/templates/kubecost-agent-secret-template.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.agent }}
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: {{ template "kubecost.agentStoreSecretName" . }}
```

Comment:
What is this secret for?

Reply:
The secret is used specifically for the agent to access S3 storage for both metrics writing and shared configuration.